release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ for details.
 
 For more information, see xref:../authentication/bound-service-account-tokens.adoc#bound-service-account-tokens[Using bound service account tokens].
 
+[id="ocp-4-4-oauth-proxy-imagestream"]
+==== The `oauth-proxy` imagestream is now available
+
+{product-title} 4.4 introduces the `oauth-proxy` imagestream for authentication integration with the cluster. You should no longer use the `oauth-proxy` image from the Red Hat Registry. You should instead use the `openshift/oauth-proxy:v4.4` imagestream, which guarantees backwards compatibility and allows you to add imagestream triggers to get critical fixes.
```

Comment:
"3rdparty authentication integration" maybe?

Reply:
We should say something about how long the v4.4 tag is available.

Reply:
> You should instead use the `openshift/oauth-proxy:v4.4` imagestream,

... if you target OpenShift v4.4 clusters and newer. The v4.4 tag will be available in at least v4.5, v4.6 and v4.7 without breaking changes. It guarantees backwards compatibility ....